tests/RobotFramework/tests/cumulocity/misc/custom_bootstrap.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Resource    ../../../resources/common.resource
+
+Library    Cumulocity
+Library    ThinEdgeIO
+Library    DateTime
+
+Test Teardown    Get Logs
+
+*** Test Cases ***
+
+test connect after starting all services
+    ${DEVICE_SN}=    Setup    skip_bootstrap=True
+    Execute Command    test -f ./bootstrap.sh && ./bootstrap.sh --no-connect || true
+    Execute Command    systemctl start mosquitto
+    Execute Command    systemctl start tedge-agent
+    Execute Command    systemctl start tedge-mapper-c8y
+    Execute Command    test -f ./bootstrap.sh && ./bootstrap.sh --no-install --no-secure || true
+    Device Should Exist    ${DEVICE_SN}
+
+    # FIXME: Assert that there are no child devices present.
+    # Cumulocity.Device Should Have A Child Devices    ""
```

Comment:
Just don't provide any arguments then it will work, however I'll create a new keyword as it reads funny...

```suggestion
    Cumulocity.Device Should Have A Child Devices
```

Reply:
New keyword `Device Should Not Have Any Child Devices` to assert that there are no child devices.

**file: tests/RobotFramework/requirements/requirements.txt**

```
robotframework-c8y @ git+https://github.com/reubenmiller/robotframework-c8y.git@0.31.1
```

Example

```
Cumulocity.Device Should Exist    ${DEVICE_SN}
Cumulocity.Device Should Not Have Any Child Devices
```